build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/InternalBwcGitPlugin.java

```diff
@@ -68,7 +68,7 @@ public void apply(Project project) {
 
         TaskContainer tasks = project.getTasks();
         TaskProvider<LoggedExec> createCloneTaskProvider = tasks.register("createClone", LoggedExec.class, createClone -> {
-            createClone.onlyIf(task -> this.gitExtension.getCheckoutDir().get().exists() == false);
+            createClone.onlyIf("git checkout dir does not exist", task -> this.gitExtension.getCheckoutDir().get().exists() == false);
```

Comment:
I'd try and avoid any negatives in these phrases, since as I mentioned before, they are going to get suffixed with "not satisfied" so you end up with something awkward like "'git checkout dir does not exist' not satisfied" and double negatives hurt my brain. Perhaps something like "git checkout dir missing".

Reply:
🧠  already too late here for double negatives